lib/matplotlib/contour.py

```diff
@@ -1386,7 +1386,7 @@ def _process_args(self, *args, corner_mask=None, algorithm=None, **kwargs):
         """
         Process args and kwargs.
         """
-        if isinstance(args[0], QuadContourSet):
+        if len(args) and isinstance(args[0], QuadContourSet):
```

Comment:
Just `if args` should work too.

Reply:
Yeah, you are right. I'll change it. Thanks!